params:
  image_params:
    image_size: 640
  ros_topics:
    raw_image_topic: "/orb_slam3/keyframe_image"
    segmented_image_topic: "/camera/color/image_segment"
    segmented_image_vis: "/camera/color/image_segment_vis"
  model_params:
    model_name: "SegFormer"
    conf: 0.4
